elliottlib/cli/find_builds_cli.py

```diff
@@ -202,9 +202,7 @@ def _fetch_builds_by_kind_rpm(tag_pv_map):
     return rpm_tuple
 
 
-def _attached_to_open_erratum_with_correct_pv(kind, results, errata):
-    if kind != "image":
```

Comment:
This might be a potential problem IIRC since this filter with erratum thing used to be "only images", last time I was trying to remove this caused some problems.

Reply:
Currently what this function does is filtering builds that has been shipped or other advisories with the same version (by reading the ART comment) out of the list to attach. I don't think there will be an issue with it. But better to confirm with @sosiouxme

Reply:
Yeah, it was just my vague memories of having a problem on this part, maybe it's gone now. @sosiouxme do you know what was the reason why we use this errata filter only for images, not RPMs as well?

Reply:
I cannot judge whether removing this test will break an edge condition. It is however a cheap test. If we were to treat it as an `assert`, it'd probably more easy to live with it.

I'd say to undo the removal of `_attached_to_open_erratum_with_correct_pv` in this PR, so we can get it merged.

Maybe this function can log something when it takes an action so we can find  circumstances if we want to?

Reply:
@joepvd The point of this PR is not attaching shipped release RPMs to an advisory. If we remove this check, this PR will be meaningless.

Reply:
really sorry for not looking at this sooner.

@shiywang this condition is a recent addition of yours from https://github.com/openshift/elliott/commit/df8be4a9fa3d9af9b3b30af1dc3383972aa7076a#diff-0e3c3290b54e518e2da0eb3dc26df8ceR212
(part of https://github.com/openshift/elliott/pull/108 recently merged).

I don't think it makes sense (I also don't think the original name for this made sense, it was removing all the things _not_ attached to an open erratum, so @vfreex 's name change here seems good). we don't want to re-attach what's already been attached/shipped for the same version, whether it's an image or rpm. this _may_ have been what [pulled in already-attached rpms on 4.3.0](https://issues.redhat.com/browse/OCPPLAN-2622?focusedCommentId=13947211&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13947211) (though it's hard to tell which thing went wrong there).